bigquery/quickstart.rb

```diff
@@ -0,0 +1,34 @@
+# Copyright 2016 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START bigquery_quickstart]
+# Imports the Google Cloud client library
+require "google/cloud"
+
+# Your Google Cloud Platform project ID
+project_id = "YOUR_PROJECT_ID"
+
+# Instantiates a client
+gcloud = Google::Cloud.new project_id
```

Comment:
Because we already have a comment with the wording *"Instantiates a client"*, I don't think we need to call the BigQuery client `bigquery_client`.  I really like the clarity this adds but no other samples use this working.

All samples on the client library website use `bigquery` not `bigquery_client`:
https://googlecloudplatform.github.io/google-cloud-ruby/#/docs/google-cloud/v0.20.1/google/cloud/bigquery

Also, we use `bigquery` in our code samples and never use a `_client` suffix:
https://github.com/GoogleCloudPlatform/ruby-docs-samples/blob/2016.09.29/bigquery/tables.rb#L24

Let's update the syntax of the quickstarts to look more idiomatic, like so:

```ruby
# Instantiates a client
gcloud   = Google::Cloud.new project_id
bigquery = gcloud.bigquery
```